server/tailsql/options.go

```diff
@@ -399,20 +407,11 @@ func (h *dbHandle) Tx(ctx context.Context, f func(context.Context, *sql.Tx) erro
 	// safe, but to prevent the handle from being swapped (and the database
 	// closed) while connections are in-flight.
 	//
-	// For our uses we could mark transactions ReadOnly, but not all database
-	// drivers support that option (notably Snowflake does not).
```

Comment:
not relevant anymore?

Reply:
Correct—already removed in one of the later commits.